### PR TITLE
Rebase #13757 to master

### PR DESCRIPTION
**Pull request by ymjiang**
## Description ##
This is a rebase version of [https://github.com/apache/incubator-mxnet/pull/13757](url)

## Details ##

https://github.com/apache/incubator-mxnet/blob/a38278ddebfcc9459d64237086cd7977ec20c70e/example/image-classification/train_imagenet.py#L42

When I try to train imagenet with this line commented, the train-accuracy reaches 99% while the validation-accuracy is only less than 50% (single machine, 8 GPUs, global batchsize=2048, Resnet50, fp32). Absolutely this is overfitting.

Then I uncomment this line and try again with the same experiment settings. This time both train and validation accuracy converge to about 66%, which looks like normal result.

Thus, it seems that this data augmentation is pretty important for ImageNet training. Perhaps it will be better to uncomment this as default, so that future developers won't get confused by the overfitting issue.

My commits enable data-augmentation with command-line argument.

### CODE REVIEW

**Comment by piyushghai** [comment]
@ymjiang Can you make the PR title a bit more descriptive please ? 
 @mxnet-label-bot Add [pr-awaiting-review]

**Comment by Roshrini** [comment]
@ymjiang Can you please retrigger CI build?

**Comment by ymjiang** [comment]
@Roshrini Hi, I closed the issue and reopend it. Is that the correct way to re-trigger CI build?

**Comment by roywei** [comment]
@mxnet-label-bot add [pr-awaiting-merge]

**Comment by wkcn** [comment]
Thanks for your contribution!

File: example/image-classification/train_imagenet.py [hunk 1]
```diff
@@ -56,6 +54,8 @@ def set_imagenet_aug(aug):
         dtype            = 'float32'
     )
     args = parser.parse_args()
```
Comment (shuo-ouyang) [inline example/image-classification/train_imagenet.py:56]: Maybe we should rearrange line 56-58? It looks like set_imagenet_aug() does nothing on args.
